docs/installation.rst

```diff
@@ -51,7 +51,8 @@ Installing from npm
 
 You can install CasperJS using `npm <http://npmjs.org/>`_::
 
-    $ npm install -g casperjs
```

Comment:
@istr I do not understand. If I understand your PR correctly for 1.1.0-beta4 (#1365) `npm install -g casperjs` should work as expected/be addressed. So there is no need for my PR?